theseus/geometry/se3.py

```diff
@@ -97,10 +97,138 @@ def _hat_matrix_check(matrix: torch.Tensor):
 
     @staticmethod
     def exp_map(tangent_vector: torch.Tensor) -> LieGroup:
-        raise NotImplementedError
+        if tangent_vector.ndim != 2 or tangent_vector.shape[1] != 6:
+            raise ValueError("Tangent vectors of SE(3) can only be 6-D vectors.")
+
+        NEAR_ZERO_EPS = 5e-3
```

Comment:
We should consolidate all the EPS, otherwise these spread around could make debugging messy.

Reply:
This is part of #70